### PR TITLE
Print "-- data '{}'" when the data is {}

### LINKED ISSUE #30: A post with json data being an empty object doesn't print the --data '{}'
A POST without a body is not exactly the same as a POST with a '{}' body, right now empty bodies are omitted and the resulting curl doesn't have the --data property.

### PR DESCRIPTION
**Pull request by kemaleq**
Resolves #30

### CODE REVIEW

File: src/lib/CurlHelper.js [hunk 1]
```diff
@@ -78,7 +79,8 @@ export class CurlHelper {
     let url = this.getUrl();
 
     if (this.getQueryString() != "") {
-      url = (url.charAt(url.length - 1) == '/')  ? url.substr(0, url.length - 1) : url; 
+      url =
+        url.charAt(url.length - 1) == "/" ? url.substr(0, url.length - 1) : url;
```
Comment (delirius325) [inline src/lib/CurlHelper.js:83]: Please keep this variable assignement on the same line. The short-hand if statement could possibly be seperated on multiple lines however.

**Review by delirius325 — CHANGES_REQUESTED** [review]

**Comment by delirius325** [comment]
Thanks for the contribution, it's much appreciated 😄 ! Please review my request and I'll approve the changes.

File: src/lib/CurlHelper.js (hunk 1 above)
Comment (kemaleq) [reply src/lib/CurlHelper.js]: I think we're on the limit for this line ;) . My Prettier made that change, but I think you're absolutely right in this case.

**Comment by kemaleq** [comment]
> Thanks for the contribution, it's much appreciated 😄 ! Please review my request and I'll approve the changes.

My pleasure, thanks for your hard work. Very handy lib! 😄

**Review by delirius325 — APPROVED** [review]

**Pull request merged**